app/src/main/java/com/infomaniak/drive/ui/bottomSheetDialogs/DriveMaintenanceBottomSheetDialog.kt

```diff
@@ -37,14 +39,27 @@ class DriveMaintenanceBottomSheetDialog : InformationBottomSheetDialog() {
             setImageResource(R.drawable.ic_maintenance)
         }
 
-        title.text = resources.getQuantityString(R.plurals.driveMaintenanceTitle, 1, navigationArgs.driveName)
-        description.setText(R.string.driveMaintenanceDescription)
-        actionButton.apply {
-            setText(R.string.buttonClose)
-            setOnClickListener { dismiss() }
-        }
+        if (navigationArgs.isAsleep) {
+            title.text = resources.getString(R.string.maintenanceAsleepTitle, navigationArgs.driveName)
+            description.text = getString(R.string.maintenanceAsleepDescription)
+            actionButton.apply {
+                setText(R.string.buttonClose)
+                setOnClickListener { dismiss() }
+            }
```

Comment:
This is duplicated between your if branch, you can extract it like it was before